include/ginkgo/core/base/precision_dispatch.hpp

```diff
@@ -226,6 +226,111 @@ void precision_dispatch_real_complex(Function fn, const LinOp *alpha,
 }
 
 
+/**
+ * Calls the given function with each given argument LinOp
+ * converted into matrix::Dense<ValueType> as parameters.
+ *
+ * If GINKGO_MIXED_PRECISION is defined, this means that the function will be
+ * called with its dynamic type with its static type, so the (templated/generic)
```

Comment:
```suggestion
 * called with its dynamic type instead of with its static type, so the (templated/generic)
```

Reply:
I think I made a small typo there, what I meant to write is
```suggestion
 * called with its dynamic type as a static type, so the (templated/generic)
```